http-server/apache2-web-server/README.md

```diff
@@ -82,31 +93,31 @@ $ nano web/htdocs/index.html
 </html>
 ```
 
-Or Download it from this repo:
+Or download it from this repo:
 
 ```
-$ wget https://raw.githubusercontent.com/sylabs/examples/master/http-server/apache2-web-server/index.html
+$ wget -O httpd/htdocs/index.html https://raw.githubusercontent.com/sylabs/examples/master/http-server/apache2-web-server/index.html
 ```
 
-
 <br>
 
-
-To use this structure, we start up an instance binding our host path into the container:
+Now our directory map should look like this:
 
 ```
-$ singularity instance start \
- -B web/htdocs:/usr/local/apache2/htdocs \
- -B web/logs:/usr/local/apache2/logs \
- httpd.sif httpd
+~/httpd/
+ |-- htdocs/
+ |   `-- index.html
+ `-- logs/
 ```
 
 <br>
 
-*FYI:* the above command is the same as:
+## Start the instance:
+
+To use this structure, we start up an instance binding our host path into the container:
 
 ```
-$ singularity instance start -B web/htdocs:/usr/local/apache2/htdocs -B web/logs:/usr/local/apache2/logs httpd.sif httpd
+$ singularity instance start -B httpd/:/srv/httpd/ httpd.sif httpd
```

Comment:
Change `httpd/:/srv/htdocs` to be:  `htdocs/:/srv/httpd/`

If we're *in* `~/httpd/` ... then the relative path is `htdocs/`. Use `~/httpd/htdocs/` for the absolute path to work no matter where we run the image from.

Reply:
Oh I see now, this is what the directory map should look like:

```
~/httpd/     #  we are in this one
 |-- httpd/  #  this is the dir we are binding
 |   `-- htdocs/index.html
 |   `-- logs/
```

I hope that make sense now.